src/tactic/ui/tools/pipeline_wdg.py

```diff
@@ -6613,3 +6618,394 @@ def execute(self):
         pass
 
 
+
+
+class PipelineDocumentWdg(BaseRefreshWdg):
+
+    def get_document(self):
+        project_code = self.kwargs.get("project_code") or Project.get_project_code()
+
+        document = {
+            'type': 'table',
+            'content': [
+                {
+                    "type": "group",
+                    "state": "on",
+                    "group_level": 0,
+                    "title": "Workflows",
+                },
+                {
+                    "type": "group",
+                    "state": "on",
+                    "group_level": 1,
+                    "title": "Job",
+                },
+                {   
+                    "type": "sobject",
+                    "group_level": 2,
+                    "expression": "@SEARCH(sthpw/pipeline['category','Job']['@ORDER_BY','name']['project_code', '%s'])" % (project_code)
```

Comment:
why is category job?

Reply:
I think that the original PipelineListWdg has seperated pipelines by search type, which is why you are seeing Job. I think you can remove this.

Reply:
I just took the default category names for these. 

Forgot to mention that if we commit these changes, we have to input default categories for the pipelines we already have.